openmdao/docs/jb_utils.py

```diff
@@ -0,0 +1,88 @@
+import importlib
+import inspect
+from IPython.display import display, HTML, Code
+
+
+def _get_object_from_reference(reference):
+    """
+    Return object of given reference path.
+
+    Parameters
+    ----------
+    reference : str
+        Dot path of desired class.
+
+    Returns
+    -------
+    str
+        Object of the given class.
+    """
+    split = reference.split('.')
+    right = []
+    module = None
+    while split:
+        try:
+            module = importlib.import_module('.'.join(split))
+            break
+        except ModuleNotFoundError:
+            right.append(split.pop())
+    if module:
+        for entry in reversed(right):
+            module = getattr(module, entry)
+    return module
+
+def display_source(reference, hide_doc_string=False):
+    """
+    Return the source code of the given reference path.
+
+    Parameters
+    ----------
+    reference : str
+        Dot path of desired class or function.
+
+    Returns
+    -------
+    str
+        Source code of the given class or function.
+    """
+    obj = inspect.getsource(_get_object_from_reference(reference))
+
+    if hide_doc_string:
+        obj = obj.split('"""')
+        del obj[1]
+        obj = ''.join(obj)
+
+    return Code(obj, language='python')
```

Comment:
It seems like this shouild actually call `display(Code(...` rather than just returning the code object.